f3-sidecar/.gitignore

```diff
@@ -1,2 +1,4 @@
 # The default f3 database folder
 /f3-db
+# The binary
```

Comment:
Would it make sense to produce the binary in a dedicated directory, e.g., `out/`?

Reply:
We don't generate the binary by default. The path is the default output of go build . This PR is Adding it to gitignore just for convenience